dora/stdlib/Int.dora

```diff
@@ -42,6 +42,43 @@
   @internal fun unaryMinus() -> Int;
   @internal fun not() -> Int;
 
+  @internal fun countZeroBits() -> Int = self.not().countOneBits();
+  @internal fun countOneBits() -> Int {
```

Comment:
The change in internalck now allows marking methods and functions with implementations as internal.